src/Model/Completions.re

```diff
@@ -71,51 +70,3 @@ let setFilter = (filter, model) => {
   filter: Some(filter),
   filtered: filterItems(Some(filter), model.all),
 };
-
-let reduce = (model, action: Actions.t) => {
```

Comment:
These have been moved to the `CompletionStore`. Having them all in one place makes it much easier to reason about what's going on.